modules/material/themes/material/silauth/loginuserpass.twig

```diff
@@ -0,0 +1,146 @@
+        <div class="mdl-card__media white-bg margin" layout-children="column">
+          <img src="/logo.png" alt="{{ '{login:logo}'|trans({'%idpName%': idpName}) }}">
+        </div>
+
+        <div class="mdl-card__title center">
+          <h1 class="mdl-card__title-text">
+            {{ '{login:header}'|trans({'%idpName%': idpName}) }}
+          </h1>
+        </div>
+
+        <div class="mdl-card__supporting-text" layout-children="column">
+          <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+            <label for="username" class="mdl-textfield__label">
+              {{ '{login:label_username}'|trans }}
+            </label>
+
+            <input
+              type="text"
+              required
+              name="username"
+              class="mdl-textfield__input mdl-color-text--black"
+              value="{{ username|e }}"
+              {% if username is empty %} autofocus {% endif %}
+              id="username"
+            >
+          </div>
+
+          <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+            <label for="password" class="mdl-textfield__label">
+              {{ '{login:label_password}'|trans }}
+            </label>
+
+            <input
+              type="password"
+              required
+              name="password"
+              class="mdl-textfield__input mdl-color-text--black"
+              {% if not username is empty %} autofocus {% endif %}
+              id="password"
+            >
+          </div>
+        </div>
+
+        {% if errorCode ?? '' == 'WRONGUSERPASS' %}
+          {% set errorMessageKey = errorparams[1] ?? '{login:error_wronguserpass}'|trans %}
+          {% set errorMessageTokens = errorparams[2] ?? null %}
+          {% set message = errorMessageKey|trans({errorMessageKey, errorMessageTokens}) %}
+          <p class="mdl-card__supporting-text mdl-color-text--red error">
+            <i class="material-icons">error</i>
+
+            <span class="mdl-textfield mdl-typography--caption">
+              {{ message|e }}
+            </span>
+          </p>
+
+          <script>
+            ga('send', 'event', 'error',{{ errorCode|e(js) }}, 'message',{{ message|e(js) }});
+          </script>
+        {% endif %}
+
+        <div class="mdl-card__actions" layout-children="row">
+          {% if forgotPasswordUrl is defined %}
+            <a
+              href="{{ forgotPasswordUrl|e }}"
+              target="_blank"
+              class="mdl-button mdl-button--colored mdl-typography--caption"
+            >
+              {{ '{login:forgot}'|trans }}
+            </a>
+          {% endif %}
+
+          <span flex></span>
+
+          <button class="mdl-button mdl-button--colored mdl-button--raised">
+            {{ '{login:button_login}'|trans }}
+          </button>
+        </div>
+
+        <section layout-children="row" child-spacing="space-around">
+          {% if helpCenterUrl is defined %}
```

Comment:
```suggestion
          {% if helpCenterUrl is not empty %}
```

Reply:
I'll add the `not empty` but the `defined` is necessary.